easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.3.2-foss-2016b.eb

```diff
@@ -0,0 +1,26 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
```

Comment:
Please add yourself as a contributor when you're building on others' work, rather than removing their attributions.